Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -1885,6 +1916,8 @@ class AnimatedTracking extends Animated {
 type CompositeAnimation = {
   start: (callback?: ?EndCallback) => void,
   stop: () => void,
+  reset: () => void,
+  startNativeLoop: (iterations?: ?number) => void,
```

Comment:
Yea I think having reset as public API is fine as it could be useful by it's own.